examples/test_equations.py

```diff
@@ -267,36 +339,40 @@
     opt_jac = eq_x1_hypo_x0_optimize_jac(x, ix0, iE, sx1, numpy.array(sz), sx0[ix0], sx0cr, sr, sy1, sIext1, sK, sw)
-    print "opt_jac: ", opt_jac
+    logger.info("\nopt_jac = " + str(opt_jac))
 
     x1EQopt, x0solopt = eq_x1_hypo_x0_optimize(ix0, iE, x1eq, zeq, x0[ix0], x0cr, r, yc, Iext1, K, w)
-    print "Solution with optimization:"
-    print "x1EQ: ", x1EQopt
-    print "x0sol: ", x0solopt
+    logger.info("\nSolution with optimization:")
+    logger.info("\nx1EQ = " + str(x1EQopt))
+    logger.info("\nx0sol = " + str(x0solopt))
 
     x1EQlinTaylor, x0sollinTaylor = eq_x1_hypo_x0_linTaylor(ix0, iE, x1eq, zeq, x0[ix0], x0cr, r, yc, Iext1, K, w)
-    print "Solution with linear Taylor approximation:"
-    print "x1EQ: ", x1EQlinTaylor
-    print "x0sol: ", x0sollinTaylor
+    logger.info("\nSolution with linear Taylor approximation:")
+    logger.info("\nx1EQ = " + str(x1EQlinTaylor))
+    logger.info("\nx0sol = " + str(x0sollinTaylor))
 
-    print "\nTest calc_fz_jac_square_taylor"
-    fz_jac_square_taylor = calc_fz_jac_square_taylor(numpy.array(sz), syc, sIext1, sK, sw, tau1=1.0, tau0=1.0)
-    print "fz_jac_square_taylor:"
-    print fz_jac_square_taylor
-    print calc_fz_jac_square_taylor(z, yc, Iext1, K, w, tau1=1.0, tau0=1.0)
+    # ------------------------------------------------------------------------------------------------------------------
+
+    logger.info("\n\nTest calc_fz_jac_square_taylor...")
 
+    fz_jac_square_taylor = calc_fz_jac_square_taylor(numpy.array(sz), syc, sIext1, sK, sw, tau1=1.0, tau0=1.0)
     lfz_jac_square_taylor, sfz_jac_square_taylor, v = symbol_calc_fz_jac_square_taylor(n)
     sfz_jac_square_taylor = Matrix(sfz_jac_square_taylor).reshape(n, n)
-    print "fz_jac_square_taylor:"
+
+    logger.info("\nfz_jac_square_taylor equation:" + str(fz_jac_square_taylor))
+    logger.info("\nsymbolic fz_jac_square_taylor equation:")
     for iv in range(n):
         for jv in range(n):
             sfz_jac_square_taylor[iv, jv] = sfz_jac_square_taylor[iv, jv].subs([(v["x_taylor"][jv], x1sq[jv]),
                                                                                 (v["a"][jv], 1.0), (v["b"][jv], -2.0),
                                                                                 (v["tau1"][iv], 1.0),
                                                                                 (v["tau0"][iv], 1.0)])
-        print sfz_jac_square_taylor[iv, :]
-    print lfz_jac_square_taylor(zeq, yc, Iext1, K, w, a, b2, tau1, tau0, x1sq)
+        logger.info("\n" + str(sfz_jac_square_taylor[iv, :]))
+
+    logger.info("\nfz_jac_square_taylor = " + str(calc_fz_jac_square_taylor(z, yc, Iext1, K, w, tau1=1.0, tau0=1.0)))
+    logger.info("\nsymbolic fz_jac_square_taylor = " +
+                str(lfz_jac_square_taylor(zeq, yc, Iext1, K, w, a, b2, tau1, tau0, x1sq)))
 
-    print "This is the end..."
\ No newline at end of file
+    logger.info("\nThis is the end...")
```

Comment:
Such messages look like unfinished or non-pro code.
Better write something like "Equation XYZ finished".
It might be "the end" of this function/file, but it could be used in a larger stack, where other calls are following, thus not easy to identify what ends here.